common/source/host/mmd_device.h

```diff
@@ -55,9 +55,9 @@
 
 #define KERNEL_SW_RESET_BASE (AOCL_MMD_KERNEL + 0x30)
 
-#define N6001_PCI_OCL_BSP_AFU_ID "51ED2F4A-FEA2-4261-A595-918500575509"
-//#define N6001_SVM_OCL_BSP_AFU_ID "5D9FEF7B-C491-4DCE-95FC-F979F6F061BE"
-#define D5005_PCI_OCL_BSP_AFU_ID "3BF773B0-4D46-44D5-9067-C884DEEF8C33"
+#define N6001_PCI_ASP_AFU_ID "51ED2F4A-FEA2-4261-A595-918500575509"
```

Comment:
Are new/different macros required ? Can PCI_ASP_AFU_ID be used?

Reply:
you need to compare it with something during runtime, that PCI_ASP_AFU_ID is set during compile time , we need to find out if its n6001 or d5005 at runtime, but i guess i can set another definition saying its n6001 or d5005 at compile time and use it in runtime.

Reply:
removed ID s from code and added boardtype in cmake file, that way we dont need ID's in code to compare